Source/WebCore/bindings/js/JSAudioWorkletGlobalScopeCustom.cpp

```diff
@@ -36,6 +37,8 @@ namespace WebCore {
 template<typename Visitor>
 void JSAudioWorkletGlobalScope::visitAdditionalChildren(Visitor& visitor)
 {
+    addWebCoreOpaqueRoot(visitor, wrapped());
```

Comment:
For extra safety, I think we should do:
```
addWebCoreOpaqueRoot(visitor, static_cast<ScriptExecutionContext&>(wrapped()));
```
Since opaque roots work as `void*`, they lose all typing information and casting to the base class may be important in cases where the pointer value ends up being different (though likely not the case here).

This would also be consistent with what we do for Document:
```
template<typename Visitor>
void JSDocument::visitAdditionalChildren(Visitor& visitor)
{
    addWebCoreOpaqueRoot(visitor, static_cast<ScriptExecutionContext&>(wrapped()));
}
```